build/azure-pipelines/linux/sql-product-build-linux.yml

```diff
@@ -159,6 +159,45 @@ steps:
       yarn gulp vscode-linux-x64-build-rpm
     displayName: Build Rpm
 
+  - task: UseDotNet@2
+    displayName: 'Use .NET Core sdk'
```

Comment:
I would suggest changing this to something more descriptive such as `Install .NET Core SDK for Signing` so viewers can tell WHY it's being installed.